packages/admin-panel/cypress/support/commands.js

```diff
@@ -4,11 +4,25 @@
  */
 
 import '@testing-library/cypress/add-commands';
+import 'cypress-file-upload';
 
 Cypress.Commands.add('login', ({ email, password }) => {
   cy.visit('/login');
-
   cy.findByPlaceholderText(/email/i).type(email);
   cy.findByPlaceholderText(/password/i).type(password, { log: false });
   cy.findByText('Login to your account').click();
 });
+
+Cypress.Commands.add('selectIntoTextBox', (labelText, inputText) => {
+  cy.findByLabelText(lableText).type(inputText).type('{downarrow}').type('{enter}');
```

Comment:
Sorry, this may have been a bit confusing - I suggested a change on one line, but the variable is used in other occurrences as well.

For code changes like that, most of the time it's safer to change the code in your editor rather than apply the suggestion from the review in GitHub. You may find the **Rename** command in VSCode (default shortcut: F2) handy here!

```suggestion
  cy.findByLabelText(labelText).type(inputText).type('{downarrow}').type('{enter}');
```

Similarly for `selectDropDownValue` below